src/scripts/site.js

```diff
@@ -1219,106 +1208,87 @@ var AppInfo = {};
 
     function onAppReady(browser) {
         console.log("Begin onAppReady");
-        var deps = [];
-        var isBackgroundSync = -1 !== self.location.href.toString().toLowerCase().indexOf("start=backgroundsync");
-        var isInBackground = isBackgroundSync;
-        deps.push("apphost");
 
-        if (!isInBackground) {
-            deps.push("appRouter");
-            deps.push("scripts/themeloader");
+        var isInBackground = -1 !== self.location.href.toString().toLowerCase().indexOf("start=backgroundsync");
 
-            if (browser.iOS) {
-                deps.push("css!devices/ios/ios.css");
-            }
-
-            deps.push("libraryMenu");
-        }
+        window.Emby = {};
 
         console.log("onAppReady - loading dependencies");
 
-        require(deps, function (appHost, pageObjects) {
-            if (console.log("Loaded dependencies in onAppReady"), window.Emby = {}, isBackgroundSync) {
-                return void syncNow();
+        if (isInBackground) {
+            syncNow();
```

Comment:
so you're no longer loading `apphost` in background, why?

Reply:
That's something i didn't assume it would load things into other modules. But when i tested this, it all worked as expected. Looking in more detail into the code, it loads the appHost before reaching this block of code, when it creates the connectionManager. Once the module loads one time, even if you require it a second time, it will not load a second time, it will only return the first load. If we can assume the load is always made when defining a connectionManager, the changes i made for this block of code is working as intended, because you only need the instance of the appHost module when it's not in background.

But I could add a single require before this condition, forcing the load for appHost, just in case in the event of not loading it before this happens.

Tell me what you think.

Reply:
Even if you don't require apphost anywhere, it will load on the define definition of apphost. Ofc each modules needs to be loaded as required, as all define will be dropped in the end of this conversion to webpack

Reply:
My assumptions here are that `apphost` should be already loaded here, but can we make an assert or something to check it?

Reply:
i am not sure. we could add a flag to window object to signal that appHost is loaded, or we could force a require for appHost. Both options are valid, what do you prefer?

Reply:
I made a require before all that to ensure appHost is loaded for all function scope. Check if it's ok for you